components/Map/Map.jsx

```diff
@@ -204,6 +222,45 @@ class Map extends React.Component {
     //   });
     // }
     this.map.on('load', () => {
+      // grab the Zoom Out button of the Mapbox zoom controls
+      const zoomOutControl = document.querySelector(
+        '.mapboxgl-ctrl-zoom-out'
+      );
+
+      // use state to control tooltip's visibility
+      let showZoomTooltip = false;
+
+      // if zoom controls aren't limited, add the 'Zoom out' title back
+      if (!showZoomTooltip) {
+        zoomOutControl.title = 'Zoom out';
+      }
+
+      // render the zoomtooltip component
+      const renderZoomTooltip = () => {
+        ReactDOM.render(
+          <ZoomTooltip show={showZoomTooltip} />,
+          zoomOutControl
+        );
+      };
+
+      // show the zoomtooltip on hover if the map is locked onto an ncLayer
+      const handleMouseEnter = () => {
+        if (this.state.filterGeo) {
```

Comment:
could we additionally check to see if we are at the `minZoom` level as an additional check? My hunch is that design will prefer we only render the tooltip if we are hovering over zoom out _and_ we are at max zoom, but I could be convinced either way.

Reply:
From what I understand, `filterGeo` should only be truthy if we are at the `minZoom` level. With the change you mention before this one, I don't think changing this will be necessary.

Reply:
I could be wrong, but I think `filterGeo` is insufficient for checking `minZoom`. To demonstrate, I've performed the following steps:
1. select an NC
2.  manually zoom in a little (I use scroll wheel here)
3.  hover over zoom out button
4. confirm that tooltip appears, despite us _not_ being at minZoom

<details><summary>GIF: reproducing the behavior above:</summary>
<p>

![zoom-out-hover-info](https://github.com/user-attachments/assets/27b6d0a6-79d9-4e86-a507-e18d3e35495a)


</p>
</details>

Reply:
@ryanfchase @traycn I misunderstood what behavior was desired here. I have fixed this issue!

### Now, the behavior is:

1. select a NC
    1. map auto-zooms onto it
    1. zoom out control shows disabled
1. hover over zoom out
    1. mouse cursor is `not-allowed`
    1. custom tooltip appears 
1. zoom in extra (I also used scroll wheel)
1. hover over zoom out again
    1. mouse cursor remains normal
    1. custom tooltip does not appear
1. zoom out back to current minZoom level with the NC selected
    1. control goes disabled again
1. hover over zoom out again
    1. custom tooltip appears